src/prep.c

```diff
@@ -409,7 +409,6 @@ static GameLoopResult PlayerSelectionUpdate(GameLoopData *data, LoopRunner *l)
 
 	// Menu input
 	int idx = 0;
-	const bool useMenuCmd = GetNumPlayers(PLAYER_ANY, true, true);
```

Comment:
this should not be removed; it is to make sure default  menu commands (arrow keys) are usable in menus regardless of what player one key binds are
instead this bool should depend on whether there are multiple local non-AI players

Reply:
Gotcha, I'll try another approach.